signac/core/h5store.py

```diff
@@ -55,12 +55,22 @@ def _requires_tables():
 logger = logging.getLogger(__name__)
 
 
+class ClosedFileError(RuntimeError):
+    "Raised when trying to access a closed group."
+    pass
+
+
 def _h5set(file, grp, key, value, path=None):
     """Set a key in an h5py container, recursively converting Mappings to h5py
     groups and transparently handling None."""
+    import h5py
     import numpy    # h5py depends on numpy, so this is safe.
     path = path + '/' + key if path else key
```

Comment:
I just had a quick question, I've seen this construct pop up a few times in the h5store code, is this path refer to a file path? Should `os.path` be used to ensure the right direction of slash is used?

Reply:
@mikemhenry It's not a platform-dependent implementation in my understanding, the HDF5 standard uses a forward slash for separations. Source: https://support.hdfgroup.org/HDF5/Tutor/crtgrpar.html

Reply:
Cool, I've only used HDF5 with pandas and I couldn't quite work out if this was a literal file system path or a path in the HDF5 api..